#54: Fix wait when replacing file in bucket (#65)
exasol/exasol-testcontainers: doc/design.md

# Design Decisions

## How do we Validate That Objects on BucketFS are Ready to Use?

BucketFS is a distributed filesystem with an HTTP interface. When users upload objects to a Bucket, it takes a while until they are really usable.

> The reason are various asynchronous processes an object has to go through, like node synchronization and extraction of archives.

Author: I think the verb relates to processes. At least in German in this sentence you need plural. I am guessing in English too. I am going to rephrase the sentence though.

> The reason are various asynchronous processes an object has to go through, like node synchronization and extraction of archives.

Author: ```suggestion
This is caused by various asynchronous processes an object has to go through, like node synchronization and extraction of archives.
```

> The reason are various asynchronous processes an object has to go through, like node synchronization and extraction of archives.

Reviewer: The verb belongs to 'The reason'. So it's 'is' no matter what you have after
